gallery/src/components/NxNexusPageHeader/NexusPageHeaderExample.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {faSave, faCog } from '@fortawesome/free-solid-svg-icons';
+
+import { NxButton, NxNexusPageHeader, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+const NexusPageHeaderExample = () => {
+  const links = [{
+    name: 'NxButton',
+    href: '#/pages/NxButton'
+  }, {
+    name: 'NxAlert',
+    href: '#/pages/NxAlert',
+    current: true
+  }];
+
+  function onButtonClick() {
+    alert('Click');
+  }
+
+  return (
+    <NxNexusPageHeader productInfo={{ name: 'Test Product', version: 'Fancy code name 1.0.0' }}
+                       homeLink="#"
+                       links={links}>
+      <div className="nx-page-header__extra-content">
```

Comment:
You're ending up with two `.nx-page-header__extra-content` (nested) in the DOM

Reply:
> You're ending up with two `.nx-page-header__extra-content` (nested) in the DOM

Cleaned up here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/dabba96716d22cd05699a15630b04bace10844c8